packages/react-card/README.md

```diff
@@ -1,4 +1,4 @@
-# @fluentui/react-cards
+# @fluentui/react-card
 
 **Card components for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
```

Comment:
Can you update the rest of this readme content so it mirrors the template readme? (scripts/create-package/plop-templates-react/README.md) Most of the existing content is about the old Card.